src/graphics/Graphviz.jl

```diff
@@ -108,6 +108,10 @@ Edge(path::Vector{String}, attrs::AbstractDict) = Edge(map(NodeID, path), attrs)
 Edge(path::Vector{String}; attrs...) = Edge(map(NodeID, path), attrs)
 Edge(path::Vararg{String}; attrs...) = Edge(map(NodeID, collect(path)), attrs)
 
+@auto_hash_equals struct Title <: Statement
```

Comment:
To adhere better to the Graphviz spec, which allows labels in subgraphs too, I would replace the `Title` struct with

```julia
Base.@kwdef struct Label <: Statement
  labelloc::String = ""
  label::String = ""
end
```
We can still use the keyword argument `title` below.

Reply:
Graphviz [docs](https://graphviz.org/docs/attrs/labelloc/) say only "t" and "b" are allowed for graphs and clusters, so could `labelloc` instead be a bool `top` that defaults to `true`?

Reply:
That's fine with me, but one reason to prefer the string is that nothing will break if Graphviz adds other allowed options for `labelloc`.